### PR TITLE
Coherency QB workflow documentation

### PR DESCRIPTION
**Pull request by mmitche**

### CODE REVIEW

File: Documentation/3xReleaseShutdownTasks.md [hunk 1]
```diff
@@ -0,0 +1,391 @@
...
+    2. For a servicing release, these changes may simply be commited into an
+       existing release branch.
+2. Official builds and dependency flow begins in each repository with changes.
+3. New commits are built, generating new assets, which flow via Maestro++
+   subscriptions into other repositories.
+4. Dependency flow generates new commits, which in turn cause new builds and
+   additional dependency flow.
+5. Repeat 3-4 until no additional changes appear and build is 'coherent'. See
+   [What is coherency and why is it important](#what-is-coherency-and-why-is-it-important)
+6. Starting with SDK (which aggregates all repos), gather a drop for the build.
```
Comment (wtgodbe) [inline Documentation/3xReleaseShutdownTasks.md]: I would change "SDK" to "core-sdk" (perhaps with a link) for clarity. Same with the line below

File: Documentation/3xReleaseShutdownTasks.md [hunk 2]
```diff
@@ -0,0 +1,391 @@
...
+reaction to breaking changes, and desired subscription update frequencies. As
+those repositories then flow elsewhere and eventually coalesce under
+dotnet/core-sdk, there may be a number of different versions of
+Microsoft.NETCore.App that have been transitively referenced throughout the
+graph. This is called **incoherency**. When only a single version of each
+product dependency is referenced throughout the dependency graph, the graph is
+**coherent**.
+
+***What kinds of problems of does incoherency cause?*** Incoherency represents a
+*possible* error state. For an examplem let's take a look at
```
Comment (wtgodbe) [inline Documentation/3xReleaseShutdownTasks.md]: `examplem` -> `example`

File: Documentation/3xReleaseShutdownTasks.md [hunk 3]
```diff
@@ -0,0 +1,391 @@
...
+    - Are any repositories far out of date in the Dev channel? This should
+      generally be watched by .NET Core Tactics, but other pairs of eyes is
+      always useful.
+    - Is overall dependency flow health okay for the Dev channel? Have product
+      dependency cycles been accidentally introduced?
+
+## Coherency QB workflow
+
+The coherency QB is responsible for the following:
+- **Verifying** the dependency flow is correct and will produced the desired
```
Comment (wtgodbe) [inline Documentation/3xReleaseShutdownTasks.md]: `produced` -> `produce`

File: Documentation/3xReleaseShutdownTasks.md [hunk 4]
```diff
@@ -0,0 +1,391 @@
...
+1. **Turn off core-sdk builds and/or dependency flow out of specific components to
+   avoid mixed branding** - In Azure DevOps, disable the official build pipeline
+   for dotnet/core-sdk. You may also need to disable other flow if other
+   repositories branch early. Please see below for notes.
+2. **Branch/commit for release** - Repository owners should branch and commit
+   branding updates. The Coherency QB need only monitor the status of this as
+   the core-sdk build cannot be turned back on until branding is consistent.
+3. **Turn back on disabled dependency flow if necessary** - If some repositories
+   had their dependency flow disabled because of an early branching scenario,
+   now is the time to turn these back on
```
Comment (wtgodbe) [inline Documentation/3xReleaseShutdownTasks.md]: missing period

File: Documentation/3xReleaseShutdownTasks.md [hunk 5]
```diff
@@ -0,0 +1,391 @@
...
+   of the simplicity of authentication.  There is a build definition that can create
+   this today: https://dev.azure.com/devdiv/DevDiv/_build?definitionId=9767.
+   1. Find the unified directory of shipping packages we want on the feed.  If
+      looking at the on your local machine, e.g.
+      `\\vsufile\patches\sign\NET\CORE_BUILDS\3.0.X\3.0.0\preview6\3.0.100-preview6-012264\shipping\packages`
+   2. Start the build definition using that directory using the following parameters
+      - Change the CentralBlobFeedUrl to point to an unused, isolated feed. Keep
+        the storage account the same but change the container to something new.
+        E.g.
+        https://dotnetfeed.blob.core.windows.net/**dotnet-core-preview6-012258**/index.json.
```
Comment (wtgodbe) [inline Documentation/3xReleaseShutdownTasks.md]: Were the askteriks' supposed to be for formatting? They show up literally in the .md file

**Comment by wtgodbe** [comment]
It may be useful to include an example `.png` of the flow-graph, perhaps when discussing the example of what an incoherent state could look like

File: Documentation/3xReleaseShutdownTasks.md (hunk 5 above)
Comment (mmitche) [reply Documentation/3xReleaseShutdownTasks.md]: Dang, they're supposed to bold things.

**Pull request merged**